<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>487</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ToyParser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Toy code</string>
    </property>
    <widget class="QTextEdit" name="sourceCodeText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>311</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="parseButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>390</y>
       <width>80</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Parse</string>
     </property>
    </widget>
    <widget class="QComboBox" name="reprTypeComboBox">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>392</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>binary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>decimal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hexadecimal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mnemonic</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>361</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QTextEdit" name="outputText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>341</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="copyToClipboardButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>390</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy to clipboard</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuToyParser">
    <property name="title">
     <string>ToyParser</string>
    </property>
   </widget>
   <addaction name="menuToyParser"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
